File: GithubGists/app/src/main/java/ru/avystavkin/githubgists/models/server/GistFileInfoServer.java
package ru.avystavkin.githubgists.models.server;


import com.google.gson.annotations.SerializedName;

public class GistFileInfoServer {

    @SerializedName("filename")
    private String mFileName;

    @SerializedName("content")
    private String mContent;

    public String getFileName() {
        return mFileName;
    }

    public void setFileName(String fileName) {
        this.mFileName = fileName;
    }

    public String getContent() {
        return mContent;
    }

    public void setContent(String content) {
        this.mContent = content;
    }
}

File: GithubGists/app/src/main/java/ru/avystavkin/githubgists/models/server/GistFileInfoServer.kt
package ru.avystavkin.githubgists.models.server


import com.google.gson.annotations.SerializedName

class GistFileInfoServer {

    @SerializedName("filename")
    var fileName: String? = null

    @SerializedName("content")
    var content: String? = null
}
